Dockerfile

```diff
@@ -1,13 +1,15 @@
-FROM circleci/openjdk:8
+FROM eclipse-temurin:21
```

Comment:
Are there any reasons that you adopted v21? 
Because we support from java v8 ref: https://github.com/launchableinc/cli?tab=readme-ov-file#prerequisite and we test used by java v8 https://github.com/launchableinc/cli/blob/main/.github/workflows/python-package.yml#L52

Reply:
No, let me update it. I confirmed there is a tag that the version is 8.
![Screenshot 2024-07-30 at 9 53 41](https://github.com/user-attachments/assets/a4157b17-4643-4d86-a4cd-cdb07b4d9ee7)

Reply:
https://github.com/docker-library/docs/blob/master/eclipse-temurin/README.md#shared-tags

Reply:
[130482a](https://github.com/launchableinc/cli/pull/898/commits/130482a183cd8524e8f4b92b1e59473f04f56163)